containers/go-zero/function/handler.go

```diff
@@ -59,18 +57,37 @@ type DetectionResult struct {
 	ImageBase64 string
 }
 
+var imageToUse image.Image
+var cascadeFile []byte
+
+func init() {
+	debug.SetGCPercent(-1) // Disabling automatic garbage collection.
+	image_path := os.Getenv("image_path")
+	data, err := ioutil.ReadFile(image_path)
+	if err != nil {
+		panic(err)
+	}
+	dataReader := bytes.NewBuffer(data)
+	imageToUse, _, err = image.Decode(dataReader)
+	if err != nil {
+		panic(err)
+	}
+	cascadeFile, err = ioutil.ReadFile("./data/facefinder")
+	if err != nil {
+		panic(err)
+	}
+}
+
 // Handle a serverless request
 func Handle(req http.Request) ([]byte, error) {
-	before := time.Now()
 	OldHandle()
-	output := time.Since(before).Nanoseconds()
-	return []byte(fmt.Sprintf("%v", output)), nil
+	return nil, nil
 }
 
 // Original pigo serverless handle
 func OldHandle() string {
 	var image []byte
-	fd := NewFaceDetector("./data/facefinder", 20, 2000, 0.1, 1.1, 0.18)
+	fd := NewFaceDetector(20, 2000, 0.1, 1.1, 0.18)
 	faces, err := fd.DetectFaces(imageToUse)
```

Comment:
Transforma image to use em um parâmetro do NewFaceDetector (assim como cascadeFile). Isso é fácil demais de fazer e tem um efeito positivo: Sabemos exatamente onde e como as variaveis globais (no nível de pacote) estão sendo usadas.